src/vs/workbench/browser/media/part.css

```diff
@@ -90,7 +90,7 @@
 .monaco-workbench .part.editor > .content .monaco-progress-container {
 	position: absolute;
 	left: 0;
-	top: 33px; /* at the bottom of the 35px height title container */
+	top: var(--editor-group-tab-total-height);
```

Comment:
The progress bar is not only used for the editor group title bar, but also in many other areas so we can't just change this value. It could however be overwritten just for the editor title bar for example [here](https://github.com/microsoft/vscode/blob/9dbe48372d1b3ac1abc92ff8f319d1edcefeaaef/src/vs/workbench/browser/parts/editor/media/editortitlecontrol.css).

Reply:
I like that you introduced a --editor-group-tab-total-height variable, however, you still need to remove 2px due to the height of the progress bar.